bot/extensions/eight_ball.py

```diff
@@ -0,0 +1,39 @@
+import discord
+from discord.ext.commands.cooldowns import BucketType
+from discord.ext.commands import Cog, command, cooldown
+from discord import Embed
+import random
+
+
+class EightBall(Cog):
```

Comment:
Add description to the cog also for better help menu

Reply:
It's optional but indeed it could be a nice addition.